Description: Use Debians DejaVuSans font for the fonts test.
 It is in the package ttf-dejavu-core.
Author: Tobias Hansen <tobias.han@gmx.de>

--- a/tests/test_fonts.ini
+++ b/tests/test_fonts.ini
@@ -9,7 +9,7 @@
 ttf_px3=al_load_ttf_font_stretch(ttf_filename, -24, -32, flags)
 # arguments
 bmp_filename=../examples/data/a4_font.tga
-ttf_filename=../examples/data/DejaVuSans.ttf
+ttf_filename=/usr/share/fonts/truetype/ttf-dejavu/DejaVuSans.ttf
 flags=ALLEGRO_NO_PREMULTIPLIED_ALPHA
 
 [text]
